src/components/reportabuse/content.js

```diff
@@ -0,0 +1,231 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { useAuth0 } from "@auth0/auth0-react";
+
+import InvalidIcon from "../../images/product/invalid.png"
+import { Section, Container } from "../global";
+
+const Content = () => {
+  const [url, setUrl] = useState("");
+  const [message, setMessage] = useState("");
+  const [mail, setMail] = useState("");
+  const [urlValid, setUrlValid] = useState(true);
+  const [messageValid, setMessageValid] = useState(true);
+  const { user } = useAuth0();
+
+  if (user && user.email && mail !== user.email) {
+    setMail(user.email);
```

Comment:
This means that when the user is logged in the mail will not be sent to the backend, because it doesn't pass this condition?

Reply:
No, it only means, e-mail of logged user will be prefilled into the text field.

Reply:
But then let's say someone adds some letter accidentally to it, and then uses backspace, we will get the value of the mail with letter added accidentally

Reply:
Nevermind, my mistake. I though it's the content of the field change handler function :x